### ISSUE TITLE
Processed image is rotated from portrait to landscape

### ISSUE THREAD

**Issue by zadam** [jimp-dev/jimp#1104]
# Expected Behavior

Image should preserve the original portrait orientation.

# Current Behavior

Code:

```javascript
const image = await jimp.read(buffer);

const processed = await image.getBufferAsync(jimp.MIME_JPEG);
```

Will change the orientation from portrait to landscape for this image: [IMG_5029.zip](https://github.com/oliver-moran/jimp/files/9910114/IMG_5029.zip)

- Jimp Version: 0.16.1
- Operating System: Ubuntu 22.04
- Node version: 16.18.0

**Comment by babyrusa** [comment]
I have the same issue too. Did you ever figure out why?

**Comment by hipstersmoothie** [comment]
my guess would be exif metadata isn't being re-encded. PRs welcome!

**Comment by Sarabadu** [comment]
@hipstersmoothie adder a PR trying to fix this issue #1247 , but seems like modifying the exif data also impact on the `difference` would appreciate an early review

**Comment by bobzel** [comment]
Is there a reasonably straightforward workaround?  I moved to Jimp because of all the installation dependencies of using Sharp.  However, sharp handled image orientations correctly and I don't have a good solution for Jimp.

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in [`v1.0.2`](https://github.com/jimp-dev/jimp/releases/tag/v1.0.2) :rocket:
